scripts/generate_docs.sh

```diff
@@ -1,5 +1,25 @@
 #!/usr/bin/bash
 
+RELEASE=""
+case "$1" in
+  -i|--increment)
+        CURRENT_RELEASE=$(git tag | tail -n1)
+        RELEASE="${CURRENT_RELEASE%.*}.$((${CURRENT_RELEASE##*.}+1))"
+        ;;
+  -k|--keep) # don't change anything, just keep current release
+        ;;
+  -r|--release) RELEASE=$2
+        ;;
+  *) >&2 echo "Usage: $0 [ [-k|--keep] | [-i|--increment] | [-r|--release] <release> ]"
+        exit 1
+        ;;
+esac
+
+if [ -n "$RELEASE" ] ; then
+  # Substitute version
+  sed -i "s|\(// @version \).*$|\1 $RELEASE|;" manager/manager.go
+  sed -i 's|^\(var ENGINEVERSION = "\)[^"]*\("\)$|'"\1$RELEASE\2|;" base/metrics/metrics.go
+fi
```

Comment:
It's possible someone will edit only manually only one place where version is filled (doc var or go var). What to add checking that these constants are equal?

Reply:
generate_docs.sh --get-version will show if the version differ
I've also added a warning comment to not edit this value